pkg/pillar/types/zedroutertypes.go

```diff
@@ -2872,3 +2904,25 @@ func (status OnboardingStatus) LogDelete(logBase *base.LogObject) {
 func (status OnboardingStatus) LogKey() string {
 	return string(base.OnboardingStatusLogType) + "-" + status.Key()
 }
+
+// AppNumber :
+// PS. Any change to AppNumMax, must be
+// reflected in the BitMap Size(8 bytes)
+const (
+	AppNumStart   = 0   // DhcpRange start
+	AppNumEnd     = 253 // DhcpRange end
+	AppNumMax     = 254 // Max Count
+	AppNumBridge  = 255 // For Bridge
+	AppNumInvalid = 256
+)
+
+// NetworkInstanceAppAddressMap :
+type NetworkInstanceAppAddressMap struct {
+	MapKey string
```

Comment:
Isn't this a combination of two UUIDs (network UUID + app UUID)? Or is it network instance UUID plus an appNum? In any case, the type should have the different parts. The Key() function will then combine it into a string for pubsub.

Reply:
Sure done.